src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    &__item {
+      margin: 0 10px;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
```

Comment:
In other styles there are mostly `5,10,15,50` values to be used for a `padding` or `margin`. 23 could say there is something wrong, and you have to pay attention on consistency of your paddings / margins

Reply:
You could also create `variables.scss` files with all the variables that are to be consistent and to be often used in styling files.